reference/upgrade/README.md

```diff
@@ -72,6 +72,8 @@ As we are aiming to always provide you more control and flexibility on how to de
 
 The `web` key is now a set of `locations` where you can define very precisely the behavior of each URL prefix.
 
+Note, we no longer move your application from "/" to "public/" automatically if the new format is adopted.
```

Comment:
... well it would be a lot more complicated, actually:

```sh
rsync --archive --remove-source-files --exclude=public --exclude=.git . public
# and then remove empty directories somehow
```